src/main/java/space/npstr/magma/processing/PacketProvider.java

```diff
@@ -94,19 +115,17 @@ public DatagramPacket getNextPacket(final boolean changeTalking) {
     }
 
     @Nullable
-    private DatagramPacket buildNextPacket(final boolean changeTalking) {
+    private ByteBuffer buildNextPacket(final boolean changeTalking) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. ([squid:S3776](https://sonarcloud.io/organizations/napstr-github/rules#rule_key=squid%3AS3776))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=space.npstr.magma%3Amagma&pullRequest=10&issues=AWT2H3KNvlEfaeH0cxYB&open=AWT2H3KNvlEfaeH0cxYB)

[ISSUE_KEY]: AWT2H3KNvlEfaeH0cxYB